app/domain/sub09/SubscriptionDisplayResponse.scala

```diff
@@ -46,6 +46,11 @@ case class ContactInformation (
                                 emailVerificationTimestamp: Option[String]
                               )
 
+case class vatIDs (
```

Comment:
Could you please rename the class name to VatID or VatId as case class name should be in UpperCamelCase?

Reply:
Renamed it to VatId